go/vt/vterrors/code.go

```diff
@@ -206,6 +210,11 @@ var (
 		VT14003,
 		VT14004,
 		VT14005,
+		VT15002,
```

Comment:
I'm not sure what the logic is behind error numbering, but if we are introducing `ErrorsWithNoCode`, we might want to reserve the whole 15000 series for those, and use VT14006 here instead.

Reply:
Fixed via 3cc6677c3a0cfd3c017989d2cf26d576261ebdbe